lib/src/suggestor_mixins.dart

```diff
@@ -87,28 +77,25 @@ mixin AstVisitingSuggestor<R> on AstVisitor<R> implements Suggestor {
 /// Note that this mixin provides an implementation of [generatePatches] that
 /// should not need to be overridden except for performance optimization reasons
 /// like avoiding analysis on certain files.
-mixin ElementVisitingSuggestor<R> on ElementVisitor<R> implements Suggestor {
-  final _patches = <Patch>{};
+mixin ElementVisitingSuggestor<R> on ElementVisitor<R> {
+  final patches = <Patch>[];
```

Comment:
I noticed this is a List but in `AstVisitingSuggestor`, it's a Set; is there a reason for that?

Also, are they supposed to be public?

Reply:
Yeah, it should be a set, I didn't mean to make it a list. Making it a set will de-duplicate patches, which can happen with ast/element visitors since some of the visit methods overlap due to the hierarchical nature of nodes/elements.

Reply:
And they don't need to be public, I'll change that